src/JustEat.StatsD/Buffered/Tags/SignalFxFormatter.cs

```diff
@@ -0,0 +1,17 @@
+namespace JustEat.StatsD.Buffered.Tags
+{
+    internal sealed class SignalFxFormatter : BaseTagsFormatter
```

Comment:
Similar comments as for the DataDog one.

Reply:
Renamed to SignalFxTagsFormatter and made public.